src/test/java/com/fasterxml/uuid/ext/LockedFileTest.java

```diff
@@ -0,0 +1,261 @@
+package com.fasterxml.uuid.ext;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static com.fasterxml.uuid.ext.LockedFile.READ_ERROR;
+import static org.junit.Assert.*;
+
+public class LockedFileTest
+{
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Test
+    public void constructor_givenNull_shouldThrowNullPointerException() throws IOException {
+        try {
+            new LockedFile(null);
```

Comment:
To be honest, this is kind of unit test that I think is basically worthless, and I don't want to add too many similar even for compliance purposes.

I'll tweak the PR and leave just skeletal aspect.